requirements.txt

```diff
@@ -65,9 +66,9 @@ typer==0.12.5
 typing_extensions==4.12.2
 urllib3==2.2.3
 uvicorn==0.30.6
+uvloop==0.20.0 ; sys_platform != 'win32'
```

Comment:
Yes, this is needed since windows machines like mine will not properly run the install if uvloop exists here. I think it's a MacOS thing